src/Resources/ResourceManager/Implementation/Policy/GetAzurePolicyExemption.cs

```diff
@@ -0,0 +1,197 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+namespace Microsoft.Azure.Commands.ResourceManager.Cmdlets.Implementation
+{
+    using Microsoft.Azure.Commands.ResourceManager.Cmdlets.Components;
+    using Microsoft.Azure.Commands.ResourceManager.Cmdlets.Extensions;
+    using Microsoft.Azure.Commands.ResourceManager.Common;
+
+    using Newtonsoft.Json.Linq;
+    using Policy;
+    using System.Management.Automation;
+    using System.Threading.Tasks;
+
+    /// <summary>
+    /// Gets the policy exemption.
+    /// </summary>
+    [Cmdlet(VerbsCommon.Get, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "PolicyExemption", DefaultParameterSetName = PolicyCmdletBase.DefaultParameterSet), OutputType(typeof(PsPolicyExemption))]
+    public class GetAzurePolicyExemptionCmdlet : PolicyCmdletBase
```

Comment:
Parameter set `NameParameterSet`, `__AllParameterSets` of cmdlet `Get-AzPolicyExemption` have the same mandatory parameters, and both of them are not default parameter set which may cause confusion.
I think you can remove `NameParameterSet`.

Reply:
Good point. I removed DefaultParameterSet and keep NameParameterSet as the default parameter set. The reason is to keep consistency with other az-policy comlet like Get-AzPolicySetDefinition, Get-AzPolicyDefinition.